NSS-Day-Cohort-66/rare-api-extra-medium
# Api get is fully functional

# Comments can now be queried from the database via GET, and a number of expand queries are supported

**New Supported Routes**
1. `/comments`
2. `/posts`
3. `/tags`
4. `/categories`

## STEPS TO TEST

1. Pull down the `jafgo4` branch and switch to it
2. Start/Restart your debugger/JSON-Server
3. Open Postman
4. Request `http://localhost:8000/comments` and verify you see all existing comments. Each comment should include
- a unique primary key
- a foreign key for post called post
- a foreign key for author called author
- a foreign key for user called user
- a varchar called content
- a boolean called is_owner
5. Verify response code of 201

### Request `http://localhost:8000/comments` The comments should now have three nested objects, one for each of the corresponding: posts, authors and users from the specified comments.
